packages/@react-spectrum/dialog/stories/DialogTrigger.stories.tsx

```diff
@@ -34,11 +34,51 @@ storiesOf('DialogTrigger', module)
     'type: popover',
     () => render({type: 'popover'})
   )
+  .add(
+    'type: popover, small',
+    () => render({type: 'popover', size: 'S'})
+  )
+  .add(
+    'type: popover, medium',
+    () => render({type: 'popover', size: 'M'})
+  )
+  .add(
+    'type: popover, large',
+    () => render({type: 'popover', size: 'L'})
+  )
+  .add(
```

Comment:
this one seems weird, how do we prevent this?

Reply:
I'll add some logic to prevent fullscreen and fullscreen takeover from being sent to type:popover dialogs